github-runners/terraform/base/ami.tf

```
@@ -0,0 +1,102 @@
+data "aws_ami" "ubuntu_x86" {
```

Judging by how images are named, this would be more appropriate:
```suggestion
data "aws_ami" "ubuntu_amd64" {
```
Or at least this:
```suggestion
data "aws_ami" "ubuntu_x86_64" {
```